code/modules/spells/spell_code.dm

```diff
@@ -84,6 +85,7 @@ var/list/spells = typesof(/spell) //needed for the badmin verb for now
 
 	var/obj/screen/spell/connected_button
 	var/currently_channeled = 0
+	var/gradual_casting = 0 //equals 1 while a Sp_GRADUAL spell is actively being cast
```

Comment:
It's not a standard until you standardise it 
There's absolutely no need to do this